app/resources/numismatic_citations/numismatic_citations_controller.rb

```diff
@@ -28,4 +34,8 @@ def load_numismatic_references
     def parent_resource
       @parent_resource ||= find_resource(@change_set.citation_parent_id)
     end
+
+    def load_coin
```

Comment:
Same issue as above, except we already use this from both NumismaticIssue and Coin.